faqs.html

```diff
@@ -5,29 +5,42 @@
 <div class="container white-text">
   <h1 class="text-accent-4 center">Frequently Asked Questions</h1>
   <ul class="collapsible black">
+    <li>
+      <div class="collapsible-header black clemson-orange-text">COVID-19?</div>
+      <div class="collapsible-body">
+      <span class="card-title clem"><i class="material-icons">error_outline</i></span>
+        <p>As passionate we are about cybersecurity, safety is our number one priority. We sincerly hoped that the situation around COVID-19 would improve enough to continue our event on Clemson University’s campus. However, based on local health experts and community leader recommendations, it has been determined that <strong>CUCTF 2020 will be a completely virtual event on October 3rd. </strong> We refuse to be setback from this announcement, and will use this oppertunity to: </p>
```

Comment:
A lot of passive voice in this line.

Reply:
Okay yeah I just noticed a bunch of gramma, spelling, and content issues here. I kinda glossed over it because I thought I'd already read it before but I guess not. Most of this whole line probably needs to be reworded so I'll make a suggestion later when I get a chance.

Reply:
Thank you David, ran it through grammarly to fix the issues.